src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -759,6 +764,80 @@ public void connect(String portName, int busIndex0, ModuleInst other, String oth
         }
     }
 
+    /**
+     * Connects unconnected SitePinInst inputs on the module instance to either GND
+     * or VCC (some reset input use VCC as an input and use the inverter to drive
+     * GND).
+     */
+    public void tieOffUnconnectedInputs(boolean verbose) {
+        for (Port port : getModule().getPorts()) {
+            if (port.isOutPort())
+                continue;
+
+            for (SitePinInst pin : getCorrespondingPins(port)) {
+                if (pin == null) {
+                    System.err.println("ERROR: Problem encountered with finding corresponding SitePinInst on port "
+                            + getName() + "/" + port.getName() + " skipping...");
+                }
+                if (pin != null && (pin.getNet() == null || pin.getNet().getSource() == null)) {
+                    // If a port has a RST inverter, drive it with VCC to get GND
+                    boolean connectToVCC = pin.getName().contains("RST");
```

Comment:
Actually, I'm beginning to think this is a little risky. You are connecting all module (not primitive) inputs containing `RST` to VCC which means you can't rely that all fanouts of this `RST` port will have inverters. It may even be driving LUTs.

I think it's safer to rely on the `RouterHelper.invertPossibleGndPinsToVccPins()` method (called by default during `RWRoute.preprocess()`) to make this transformation on a primitive level.